pym/bob/generators/QtCreatorGenerator.py

```diff
@@ -158,6 +158,8 @@ def qtProjectGenerator(package, argv, extra):
         help="File filter. A regex for matching additional files.")
     parser.add_argument('--exclude', default=[], action='append', dest="excludes",
             help="Package filter. A regex for excluding packages in QTCreator.")
+    parser.add_argument('--use-only', default=[], action='append', dest="useonly",
```

Comment:
I'm still not so happy with the name of the argument, even if I don't have a counterproposition. What do you think about `--include-filter` (and `--exclude-filter` instead of `--exclude`)?